CHANGELOG.md

```diff
@@ -39,7 +40,9 @@ return values of avifImageCopy() and avifImageAllocatePlanes().
 * Add imageDimensionLimit field to avifDecoder struct
 * Add autoTiling field to avifEncoder struct
 * Add AVIF_CHROMA_DOWNSAMPLING_SHARP_YUV value to avifChromaDownsampling enum
-* Add avifImageExtractExifOrientationToIrotImir()
+* Add avifImageExtractExifOrientationToIrotImir() which parses the Exif payload
```

Comment:
Nit: payload => metadata

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1131